log/handlers_test.ts

```diff
@@ -459,6 +498,7 @@ Deno.test({
 
     assertEquals((await Deno.stat(LOG_FILE)).size, 0);
     dispatchEvent(new Event("unload"));
+    dispatchEvent(new Event("unload"));
```

Comment:
Why is another `unload` event needed here?

Reply:
For test #1787, before #1787 std add an event listener in wrong way. If call `dispatchEvent(new Event("unload"));` twice, this test will fail.
I reserve this as an improve this test.